XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -57,6 +57,7 @@ public partial class VM : IComparable<VM>
         private const int DEFAULT_NUM_VIFS_ALLOWED = 7;
         private const int DEFAULT_NUM_VBDS_ALLOWED = 255;
         public const long DEFAULT_MEM_ALLOWED = 1 * Util.BINARY_TERA;
+        public const long DEFAULT_MEM_MIN_IMG_IMPORT = 256 * Util.BINARY_MEGA;

```

Comment:
What is the reason for setting the minimum memory to 256MB? Does this still cause issues?

Reply:
You just need a minimum memory value to assign to the VM that will be created (with 0 it won't start).